draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -48,6 +48,8 @@
   private static final float SENSITIVITY = 1f;
   private static final boolean DEFAULT_TOP_VIEW_RESIZE = false;
 
+  private boolean mEnableClickToExpand = true;
```

Comment:
The initial value of this variable should be `false`. This feature have to be disabled by default. Once you change this,  you have to create a method to enable this feature from DraggablePanel.